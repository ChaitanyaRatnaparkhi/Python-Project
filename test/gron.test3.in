{
  "students": [
    {"name": "Bob", "grade": 85},
    {"name": "Charlie", "grade": 92},
    {"name": "David", "grade": 78}
  ]
}