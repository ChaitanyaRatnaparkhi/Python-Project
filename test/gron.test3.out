json = {}
json.students = []
json.students[0] = {}
json.students[0].name = "Bob"
json.students[0].grade = 85
json.students[1] = {}
json.students[1].name = "Charlie"
json.students[1].grade = 92
json.students[2] = {}
json.students[2].name = "David"
json.students[2].grade = 78

